packages/terra-breakpoints/tests/jest/breakpoints.test.jsx

```diff
@@ -0,0 +1,111 @@
+import breakpoints, { activeBreakpointForSize, breakpointIsActiveForSize } from '../../src/breakpoints';
+
+describe('breakpoints', () => {
+  it('should contain the set of breakpoint values', () => {
+    expect(breakpoints).toBeDefined();
+    expect(breakpoints.tiny).toBe(0);
+    expect(breakpoints.small).toBe(544);
+    expect(breakpoints.medium).toBe(768);
+    expect(breakpoints.large).toBe(992);
+    expect(breakpoints.huge).toBe(1216);
+    expect(breakpoints.enormous).toBe(1440);
+  });
+});
+
+describe('activeBreakpointForSize', () => {
+  it('should accurately detect default size', () => {
+    expect(activeBreakpointForSize(-1)).toBeUndefined();
+    expect(activeBreakpointForSize(0)).toBe('tiny');
+  });
+
+  it('should accurately detect tiny size', () => {
+    expect(activeBreakpointForSize(544)).toBe('small');
+  });
+
+  it('should accurately detect small size', () => {
+    expect(activeBreakpointForSize(768)).toBe('medium');
+  });
+
+  it('should accurately detect medium size', () => {
+    expect(activeBreakpointForSize(992)).toBe('large');
+  });
+
+  it('should accurately detect large size', () => {
+    expect(activeBreakpointForSize(1216)).toBe('huge');
+  });
+
+  it('should accurately detect huge size', () => {
+    expect(activeBreakpointForSize(1440)).toBe('enormous');
+    expect(activeBreakpointForSize(9999)).toBe('enormous');
+  });
+});
+
+describe('breakpointIsActiveForSize', () => {
+  it('should return false for unknown breakpoint', () => {
+    expect(breakpointIsActiveForSize('test', 100)).toBe(false);
+  });
+
+  it('should return false for negative size', () => {
+    expect(breakpointIsActiveForSize('tiny', -100)).toBe(false);
+  });
+
+  it('should accurately determine default size matching', () => {
```

Comment:
Take this with a grain of salt, and it's a minor suggestion. For edge case testing, perhaps we can test a point in between the breakpoint values if others feel that adds any value (272 for example)

Reply:
I went back and forth on that; as usual, laziness prevailed. I can add some more expectations here though.